lib/entities/integrations/index.ts

```diff
@@ -15,6 +15,26 @@ export class Integrations {
       this.networkClient = networkClient;
   }
 
+  /**
+   * Retrieve an integrations matching the given identifier
+   * @param planIdentifier - Identifier of the target plan
+   * @param identifier - Id related to a specific integration
+   * @param showSensitiveData - Determines whether the returned payload has sensitive IntegrationData values attached to it.
+   * @returns A single integration or null if not found;
+   */
+  public async get(planIdentifier: string, identifier: string, showSensitiveData: boolean = false): Promise<IIntegration | undefined> {
```

Comment:
Does `IIntegration` support Slack integration now? Just wanted to double check this as previously, it was only supporting webhooks

Reply:
Yep it does! This work was done in M1 (:

Reply:
[In this PR](https://github.com/MindscapeHQ/raygunapi-js/pull/14/files)

Reply:
Lovely, thank you for checking!